perllib/FixMyStreet/Roles/CobrandEcho.pm

```diff
@@ -52,6 +52,7 @@ sub look_up_property {
         address => FixMyStreet::Template::title($result->{Description}),
         latitude => $result->{Coordinates}{GeoPoint}{Latitude},
         longitude => $result->{Coordinates}{GeoPoint}{Longitude},
+        type => $result->{PointAddressType},
```

Comment:
Sorry, actually thought of one more thing - could we only keep/use the Id here, as this is cached in the session I'd rather not store anything more than we need to.